roseWong/assignments/twopointers/lc1/lc1.py

```diff
@@ -1,37 +1,48 @@
 # Problem Statement #
-# Given an array of sorted numbers and a target sum, find a pair in the array whose sum is equal to the given target.
-
-# Write a function to return the indices of the two numbers (i.e. the pair) such that they add up to the given target.
+# Given an array of sorted numbers and a target sum,\
+# find a pair in the array whose sum is equal to the given target.
+
+# Write a function to return the indices of the two numbers (i.e. the pair)
+# such that they add up to the given target.
+# Psuedocode
```

Comment:
Write out some edge cases and test cases for them, it shows the interviewer you're thinking critically about the problem.